src/docs/cnf_content.erl

```diff
@@ -9,7 +9,7 @@
 % software distributed under the License is distributed on an
 % "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 % KIND, either express or implied.  See the License for the
-% specific language governing permissions and limitations
+% specific language governing permissions and limitations                                   
```

Comment:
Line 12 is too long: % specific language governing permissions and limitations                                   .